Sources/Subs-Admin.php

```diff
@@ -2181,4 +2181,88 @@ function emailAdmins($template, $replacements = array(), $additional_recipients
 		}
 }
 
+/**
+ * Locates the most appropriate temp directory.
+ */
```

Comment:
I posit  a new docblock

	/**
	 * Locates the most appropriate temp directory.
	 *
	 * Systems using `open_basedir` restrictions may receive errors with
	 * `sys_get_temp_dir()` as it may not be properly configured to use a
	 * safe workable directory. Since the temp directory may be readonly,
	 * check that it is writable.
	 *
	 * Directories cheked against `open_basedir`:
	 *
	 * - `sys_get_temp_dir()`
	 * - `upload_tmp_dir`
	 * - `session.save_path`
	 * - `$cachedir`
	 *
	 * @return string
	 */